src/Providers/CDLProviders.js

```
@@ -0,0 +1,76 @@
+                cdlWorker.current.onmessage = handleWorkerMessage;
+                cdlWorker.current.postMessage({
+                    code: CDL_WORKER_PROTOCOL.LOAD_FILE,
+                    args: {
+                        fileInfo: fileInfo,
+                    }
+                });
+            } catch (error) {
+                console.error("Failed to initialize worker:", error);
+            }
+        }
+    }, [fileInfo, handleWorkerMessage]);
```

Biome (1.9.4)

[error] This variable is used before its declaration.

The variable is declared here:


(lint/correctness/noInvalidUseBeforeDeclaration)